package.json

```diff
@@ -22,7 +21,7 @@
   },
   "name": "opc-lib",
   "version": "0.0.0",
-  "description": "Webcomponent opc-lib following open-wc recommendations",
+  "description": "Web component opc-lib following open-wc recommendations",
```

Comment:
Rename to `One Platform web components library following open-wc recommendations`

Reply:
Changed the text